### DISCUSSION TITLE
Regeneration based off DNA?

### DISCUSSION THREAD

**Discussion by h0rizons** [nftchef/art-engine#268] [Help + Q&A] [answered]
Is there any way to use the previous metadata / DNA to run off the same batch again? For example, if I wanted to make a "mutant" version of a collection I generated, could I redraw the layers and use the program to re-run the same batch?

**Comment by nftchef** [comment, marked as answer]
The util script `rebuildAll` is built for this case.

To regenerate/rebuild a collection with "mutations", 
Save the original _dna.json file that is output in the build directory (always back up the build directory)

then, change the artwork in your layers directory being sure the filenames remain exactly the same,
then, running the following will create a "mutated" collection
```
node utils/rebuildAll.js -s path/to/_dna.json
```
